documentation/milestones/individual_milestones/get_opportunities.md

```diff
@@ -22,9 +26,9 @@ Deploy a public API endpoint to PROD that allows users to see at least one field
 
 The launch of a public API endpoint which provides information about every grant opportunity in grants.gov represents the culmination of multiple internally focused deliverables and serves as the foundation for future development that relies on the API.
 
-By delivering this public endpoint and ensuring it remains available even when the legacy grants.gov site is experiencing planned or unplanned outages, this milestone aims to demonstrate the following value propositions:
+By delivering this public endpoint and ensuring it remains available even when the legacy grants.gov site is experiencing planned or unplanned outages, this 30k effort aims to demonstrate the following value propositions:
 
-- Exposes `beta.grants.gov/api/` as the public API for the project that technical stakeholders can access and where future endpoints and/or features will be released
+- Exposes `simpler.grants.gov/api/` as the public API for the project that technical stakeholders can access and where future endpoints and/or features will be released
```

Comment:
Nit: can we make "choose a URL for the API" a definition of done (or something), and then later just refer to the chosen URL? (Or something like that.)

Reply:
updated!